subt/zmq-subt-x4.json

```diff
@@ -115,6 +118,13 @@
       },
       "offseter": {
         "driver": "subt.offseter:Offseter"
+      },
+      "marsupial": {
+          "driver": "subt.marsupial:Marsupial",
+          "in": ["sim_time_sec"],
+          "out": ["detach"],
```

Comment:
This is confusing... mainly because it is not correct/out of date. It also gets `origin`. It just should not be here.